src/main/java/org/apache/maven/plugins/war/AbstractWarMojo.java

```diff
@@ -230,6 +230,26 @@ public abstract class AbstractWarMojo extends AbstractMojo {
     @Parameter
     private String dependentWarExcludes = StringUtils.join(Overlay.DEFAULT_EXCLUDES, ",");
 
+    /**
+     * The comma separated list of tokens to exclude from the WAR before packaging. This option may be used to implement
+     * the skinny WAR use case. Note that you can use the Java Regular Expressions engine to include and exclude
+     * specific pattern using the expression %regex[]. Hint: read the about (?!Pattern).
+     *
+     * @since 2.1-alpha-2
```

Comment:
`3.4.1`?

Reply:
This was moved, not added, so `@since` is correct here